src/instruments/src/SD/Pages/Press/Press.tsx

```diff
@@ -0,0 +1,486 @@
+import React, { FC, useState, useEffect } from 'react';
+import { render } from '@instruments/common/index';
+import { GaugeComponent, GaugeMarkerComponent, splitDecimals } from '@instruments/common/gauges';
+import { setIsEcamPage } from '@instruments/common/defaults';
+import { Triangle } from '../../Common/Shapes';
+import { PageTitle } from '../../Common/PageTitle';
+import { EcamPage } from '../../Common/EcamPage';
+import { useSimVar } from '../../../Common/simVars';
+import { SvgGroup } from '../../Common/SvgGroup';
+
+import './Press.scss';
+
+setIsEcamPage('press_page');
+
+export const PressPage: FC = () => {
+    const [cabinVs] = useSimVar('L:A32NX_PRESS_CABIN_VS', 'feet per minute', 500);
+    const [cabinAlt] = useSimVar('L:A32NX_PRESS_CABIN_ALTITUDE', 'feet', 500);
+    const [deltaPsi] = useSimVar('L:A32NX_PRESS_CABIN_DELTA_PRESSURE', 'psi', 500);
+    const [flightPhase] = useSimVar('L:A32NX_FWC_FLIGHT_PHASE', 'enum', 1000);
+
+    const deltaPress = splitDecimals(deltaPsi, '');
+    const vsx = 275;
+    const cax = 455;
+    const dpx = 110;
+    const y = 165;
+
+    const radius = 50;
+    const [systemNumber, setSystemNumber] = useState(0);
+
+    const safetyValve = 2;
+
+    useEffect(() => {
+        setSystemNumber(Math.random() < 0.5 ? 1 : 2);
+    }, []);
```

Comment:
**question:** is this intended to be random ?

Reply:
It is for now until we actually have system 1 and 2 implemented.

Reply:
You should probably add a FIXME comment somewhere because of this.

Reply:
Note that PR #5159 already created a simvar for the active press system, I suggest you read that instead of using `random()` here, as Rust implements logic on when to switch from one system to the other.

- A32NX_PRESS_ACTIVE_CPC_SYS
    - Number [0, 1, 2]
    - Indicates which cabin pressure controller is active. 0 indicates neither is active.